lerna.json

```diff
@@ -1,5 +1,19 @@
 {
+  "$schema": "https://json.schemastore.org/lerna",
+  "changelogPreset": "conventional-changelog-conventionalcommits",
+  "command": {
+    "publish": {
+      "message": "chore: publish"
+    },
+    "version": {
+      "allowBranch": "main",
+      "conventionalCommits": true,
+      "createRelease": "github"
+    }
+  },
+  "ignoreChanges": ["**/test/**", "**/*.md"],
   "npmClient": "yarn",
+  "useNx": false,
```

Comment:
looks `false` by default so we could omit this config to keep it simpler/minimal if we want, since explicitly opt-in

Reply:
I don't trust that it will always be that way.